os/Kconfig

```diff
@@ -516,6 +516,7 @@ source Kconfig.debug
 endmenu
 
 menu "System Call"
+	depends on BUILD_PROTECTED || BUILD_KERNEL
```

Comment:
This was originally existed. But when SRIB member ported loader, he removed. Is this again necessary?
https://github.com/Samsung/TizenRT/pull/2738/commits/89f4ee684d9728bc6c2fde4ce1e7190b8d2f66a4

Reply:
It is needed for flat build binary support